lib/spaceship/tunes/tunes_client.rb

```diff
@@ -1,5 +1,10 @@
 module Spaceship
   class TunesClient < Spaceship::Client
+    class << self
+      attr_accessor :login_cache_path
+    end
+    TunesClient.login_cache_path = "/tmp/spaceship_itc_login_url.txt"
```

Comment:
I'm not sure it's a good idea to add code into the class definition. Couldn't we put this into the initializer?